benchopt/utils/temp_benchmark.py

```diff
@@ -0,0 +1,83 @@
+import os
+import tempfile
+import contextlib
+
+from pathlib import Path
+
+from benchopt.benchmark import Benchmark
+
+
+# default params
+dummy_objective = """
+"""
+
+dummy_datasets = ("""
+""",)
+
+dummy_solver = ("""
+""",)
+
+# example
+datasets = [
+    (
+        "from benchopt import BaseDataset\n"
+        "class Dataset(BaseDataset):\n"
+        "    name = 'dummy-dataset'\n"
+        "    def __init__(self, param=1.):\n"
+        "        self.param = param\n"
+        "    def get_data(self):\n"
+        "        return dict(a=1, b=1)\n"
+    ),
+]
+
+solvers = [
+    (
+        "from benchopt import BaseSolver, safe_import_context\n"
```

Comment:
@Badr-MOUFAD I think triple quotes would yield more readable code blocks in this case, in order to avoid \n